Spinner/Spinner.module.less

```diff
@@ -0,0 +1,196 @@
+// Spinner.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/variables.less";
+@import "../styles/skin.less";
+
+@keyframes spinner {
+	0% {
+		opacity: 1;
+	}
+	100% {
+		opacity: 0;
```

Comment:
Does it actually fade all the way out to 0? It didn't look like it to me, but you looked at it a lot closer.